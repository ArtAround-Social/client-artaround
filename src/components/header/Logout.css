.button {
  background-color: rgba(8, 8, 102, 0.452);
  border: none;
  color: white;
  height: 70px;
  text-align: center;
}

.button:hover {
  background-color: rgba(8, 8, 102, 0.712);
}