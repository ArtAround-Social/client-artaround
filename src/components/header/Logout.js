import React from 'react';
import PropTypes from 'prop-types';
import styles from './Logout.css';

function Logout({ handleLogout }) {
  return(
    <button className={styles.button} onClick={handleLogout}>Logout</button>
  );
}

Logout.propTypes = {
  handleLogout: PropTypes.func.isRequired
};

export default Logout;
