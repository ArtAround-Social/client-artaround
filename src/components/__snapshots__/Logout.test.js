import React from 'react';
import { shallow } from 'enzyme';
import Logout from '../header/Logout';

describe('Logout component', () => {
  it('handles click', () => {
    const wrapper = shallow(
      <Logout
        handleLogout={() => { }}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
