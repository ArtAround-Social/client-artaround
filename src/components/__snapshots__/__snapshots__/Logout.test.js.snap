// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout component handles click 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Logout
</button>
`;
